reg/drone/service/battery/Parameters.0.3.uavcan

```diff
@@ -0,0 +1,71 @@
+#   - 32 most significant bits identify the vendor as:   CRC32C(ManufacturerName)
+#   - 32 least significant bits identify the battery as: CRC32C(DeviceName + ManufactureDate + SerialNumber)
+# If the battery does not support SBS, the vendor may choose arbitrary random numbers.
+# Note that these are mere recommendations. The only hard requirement for this field is to be statistically unique.
+
+uavcan.si.unit.mass.Scalar.1.0 mass
+# The total mass of the battery, including the packaging, electronics, cabling, and all auxiliary items, if any.
+# May be used for predicting the kinematic parameters of the vehicle.
+# NaN if unknown.
+
+uavcan.si.unit.electric_charge.Scalar.1.0 design_capacity
+# The maximum total charge of the pack, at 100% SoH, specified by the manufacturer.
+
+uavcan.si.unit.voltage.Scalar.1.0[2] design_cell_voltage_min_max
+# The minimum (end of discharge) and the maximum (end of charge) resting cell voltage specified by the manufacturer
+# at 100% SoH. Example: {2.8, 4.2} V. These voltages correspond to resting voltages; i.e., the stabilized voltages after
+# the discharge/charge has been terminated. Voltage below the min may be observed during discharge due to the cell's
+# internal resistance. Voltage above the max voltage may be observed during regenerative braking/charging etc due to
+# the cell's internal resistance.
+
+uavcan.si.unit.electric_current.Scalar.1.0 discharge_current
+# Recommended continuous discharge current of the battery.
+
+uavcan.si.unit.electric_current.Scalar.1.0 discharge_current_burst
+# Maximum current that may be safely discharged at least for 5 seconds.
+
+uavcan.si.unit.electric_current.Scalar.1.0 charge_current
+# Recommended continuous charge current of the battery.
+
+uavcan.si.unit.electric_current.Scalar.1.0 charge_current_fast
+# Recommended safest highest continuous charge current for the battery.
+# This may cause accelerated aging of the battery.
+
+uavcan.si.unit.electric_current.Scalar.1.0 charge_termination_threshold
+# End-of-charging current threshold. Charging may be terminated when the current falls below this threshold.
+
+uavcan.si.unit.voltage.Scalar.1.0 charge_voltage
+# The total voltage(not per-cell) that may be used by the charger to charge the battery pack.
+
+uint16 cycle_count
+# The number of charge-discharge cycles. Zero if the battery is new. May increase at runtime.
+# What constitutes a charge-discharge cycle is implementation-defined.
+
+void16
+# Was used in v0.1 for cell_count. It is now deducible from cell_voltages in Status.0.2.
+
+uint7 state_of_health_pct  # [percent]
+# The SoH of the battery, or best guess thereof; ranges from 0 (unusable) to 100 (new).
+void1
+
+Technology.0.1 technology
+# The battery technology information may be leveraged by the charger to choose the appropriate charging strategy.
+
+uavcan.si.unit.voltage.Scalar.1.0 nominal_voltage
+# The average voltage(not per-cell) the battery outputs when charged.
```

Comment:
Suggestion based on the conversation at #112

```suggestion
# The nominal voltage of the battery pack (not per-cell) as defined by the vendor.
# The Smart Battery Data Specification v1.1 uses the term "design voltage".
# E.g., a typical 22S LiCoO2 pack might report 81.4 V here.
```

Reply:
This, IMHO, is not a good way to define a standard parameter. lack of clarity.

Reply:
While we're waiting for the SBS Forum to respond, we could merge this tentatively to unblock @PetervdPerk-NXP and then go back and provide a proper definition later.

Reply:
Not sure I understand. Why would a nominal voltage (the definition of which is not known) block @PetervdPerk-NXP ?

Reply:
This is discussed in the beginning of #112